x-pack/metricbeat/module/airflow/statsd/data_test.go

```diff
@@ -0,0 +1,69 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package statsd
+
+import (
+	"fmt"
+	"net"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/elastic/beats/v7/auditbeat/core"
+	_ "github.com/elastic/beats/v7/libbeat/processors/actions"
+	"github.com/elastic/beats/v7/metricbeat/mb"
+	_ "github.com/elastic/beats/v7/x-pack/metricbeat/module/statsd/server"
+
+	mbtest "github.com/elastic/beats/v7/metricbeat/mb/testing"
+)
+
+func init() {
+	mb.Registry.SetSecondarySource(mb.NewLightModulesSource("../../../module"))
+}
+
+const (
+	STATSD_HOST = "localhost"
+	STATSD_PORT = 8126
+)
+
+func getConfig() map[string]interface{} {
+	return map[string]interface{}{
+		"module":     "airflow",
+		"metricsets": []string{"statsd"},
+		"host":       STATSD_HOST,
+		"port":       STATSD_PORT,
+		"period":     "1s",
+	}
+}
+
+func createEvent(t *testing.T) {
+	udpAddr, err := net.ResolveUDPAddr("udp", fmt.Sprintf("%s:%d", STATSD_HOST, STATSD_PORT))
```

Comment:
Nit. Check this error too. Consider using `require` instead of `assert` for this and the following assertion.
```suggestion
	udpAddr, err := net.ResolveUDPAddr("udp", fmt.Sprintf("%s:%d", STATSD_HOST, STATSD_PORT))
	require.NoError(t, err)
```